### PR TITLE
feature/625-remove-namespace

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/containers/LoginUsernameContainer.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function LoginUsernameContainer({ navigation }: { navigation: Pro
                             <TUsername
                                 value={username}
                                 onChangeText={setUsername}
-                                suffix={settings.config.accountSuffix}
+                                // suffix={settings.config.accountSuffix}
```
Comment (StefanoPitto) [inline src/containers/LoginUsernameContainer.tsx:41]: remove commented line

File: src/containers/CreateAccountUsernameContainer.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default function CreateAccountUsernameContainer({ navigation }: Props) {
                     <View style={styles.inputContainer}>
                         <TUsername
                             errorText={errorMessage}
-                            suffix={settings.config.accountSuffix}
+                            // suffix={settings.config.accountSuffix}
```
Comment (StefanoPitto) [inline src/containers/CreateAccountUsernameContainer.tsx:62]: remove commented line

File: src/containers/CreateAccountPasswordContainer.tsx [hunk 3]
```diff
@@ -181,13 +181,10 @@ export default function CreateAccountPasswordContainer({ navigation }: Props) {
...
-                            <TP size={1}>Already have an account? </TP>
-                            <TouchableOpacity onPress={() => navigation.navigate('LoginUsername')}>
-                                <TP size={1} style={styles.link}>
-                                    Login
-                                </TP>
-                            </TouchableOpacity>
+                        <View style={commonStyles.alignItemsCenter}>
+                            <TP size={1}>
+                                Already have an account? <TLink href="login">Login</TLink>
+                            </TP>
```
Comment (StefanoPitto) [inline src/containers/CreateAccountPasswordContainer.tsx:187]: Why did this change?

Comment (sadiabbasi) [reply src/containers/CreateAccountPasswordContainer.tsx:187]: I didn't change this let me check

**Review by StefanoPitto — APPROVED** [review]
Approved

**Pull request merged**